inputfiles/patches/abstractworker.kdl

```diff
@@ -0,0 +1,3 @@
+interface-mixin AbstractWorker {
+    event error type=ErrorEvent
+}
```

Comment:
Would be nice to have a final newline - if you use VSCode then "Files: Insert Final Newline" should do it for you.